run.sh

```diff
@@ -1,8 +1,30 @@
 #!/bin/sh
 
+#add environmental variable of .env to the bash script
+. ./.env
+
+#Define absolute path for the host and the serv from the relative common path
+HOST_LOG_FILE=$(realpath -P ${LOG_FILE})
+
 PATH=$(/usr/bin/getconf PATH || /bin/kill $$)
 NAME="iscsc.fr-notify-bot"
 
+RED=$(tput setaf 1)
+NORMAL=$(tput sgr0)
+
+# Creating the file and setting the right permissions:
+
+# TODO: test $? and exit if ==1
+touch $HOST_LOG_FILE
+chmod 664 $HOST_LOG_FILE
+sudo chown :root $HOST_LOG_FILE
+
+# TODO: don't run the command if there is nothinf to stop
 docker stop $(docker ps --all --quiet --filter ancestor=${NAME} --format="{{.ID}}")
+
 docker build -t ${NAME} .
-docker run --rm --interactive --tty --detach --name ${NAME} ${NAME}:latest
+echo "${RED}[!]${NORMAL} the log file (mounted in the container) owner must match the uid in the container"
```

Comment:
update this warning as it isn't true anymore: its not the owner that mlust match but the root group

Reply:
done in 046b74f